pyproject.toml

```diff
@@ -53,11 +54,7 @@ tests = [
     "dvclive[image,plots,markdown]",
     "ipython",
 ]
-dev = [
-    "dvclive[all,tests]",
-    "mypy>=1.1.1",
-    "types-PyYAML",
-]
+dev = ["dvclive[all,tests]", "mypy>=1.1.1", "types-PyYAML"]
```

Comment:
minor: do other changes to this file come from applying some linter? it should be then automated and ideally (not a blocker, no need to change anything in this PR) should be done as a separate PR. The reason is that we can't rely on some local personal settings / setup. People have different preferences, editors - we'll keep getting formatting updates.

Reply:
I reverted the formatting changes in `pyproject.toml` to introduce it in a cleaner way: https://github.com/iterative/dvclive/pull/775